client/webserver/locales/en-us.go

```diff
@@ -155,10 +155,10 @@ var EnUS = map[string]string{
 	"New Wallet Password":         "New Wallet Password",
 	"pw_change_warn":              "Note: Changing to a different wallet while having active trades might cause funds to be lost.",
 	"Show more options":           "Show more options",
-	"seed_implore_msg":            "You should carefully write down your application seed and save a copy. Should you lose access to this machine or the critical application files, the seed can be used to restore your DEX accounts and native wallets. Some older accounts cannot be restored from seed, and whether old or new, it's good practice to backup the account keys separately from the seed.",
+	"seed_implore_msg":            "Write down your application seed phrase and make a copy.",
 	"View Application Seed":       "View Application Seed",
 	"Remember my password":        "Remember my password",
-	"pw_for_seed":                 "Enter your app password to show your seed phrase. Make sure nobody else can see your screen.",
+	"pw_for_seed":                 "Application seed can be used to restore your DEX accounts and native wallets. Make sure nobody else can see your screen.",
```

Comment:
We now we have a untranslated text that's going to take forever to get retranslated.  This `pw_for_seed` message was fine, so please revert.

All that needed to be done, if anything, was to remove from `seed_implore_msg` the part: `and whether old or new, it's good practice to backup the account keys separately from the seed` part of the other message.`
Ideally do that without breaking any of the translations please.

Reply:
Okay, followed up on this in my latest commit to decrease breakage in existing translations